parser/parser.y

```diff
@@ -1695,6 +1697,20 @@ AlterTableSpec:
 			TiFlashReplica: tiflashReplicaSpec,
 		}
 	}
+|	"SET" "TIFLASH" "MODE" "NORMAL"
+	{
+		$$ = &ast.AlterTableSpec{
+			Tp:          ast.AlterTableSetTiFlashMode,
+			TiFlashMode: model.TiFlashModeNormal,
+		}
+	}
+|	"SET" "TIFLASH" "MODE" "FAST"
```

Comment:
Please also add tests to verify that the new AlterTableSpec (containing TiFlashMode) can be correctly used to recover back the "SET TIFLASH MODE XXX" statement.